modules/nf-core/rtgtools/format/tests/main.nf.test

```diff
@@ -0,0 +1,135 @@
+nextflow_process {
+
+    name "Test Process RTGTOOLS_FORMAT"
+    script "../main.nf"
+    process "RTGTOOLS_FORMAT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "rtgtools"
+    tag "rtgtools/format"
+
+    test("sarscov2 - fasta") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [id:'test'],
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true),
+                    [],
+                    []
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
```

Comment:
I think capturing the versions is missing from this snapshot

Reply:
Yes! thank you I missed that